pkg/daemon/ceph/osd/daemon.go

```diff
@@ -53,16 +53,18 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID, lvPath string,
 		logger.Errorf("failed to create config dir %s. %+v", configDir, err)
 	}
 
+	// Update LVM config at runtime
+	if err := updateLVMConfig(context, pvcBackedOSD); err != nil {
```

Comment:
I believe that with this change, `updateLVMConfig` is called in `OSD daemon pod` for both `OSD on PVC` and `OSD on Disks` flow. But in case of `OSD prepare pod`,  `updateLVMConfig` method is only called for `OSD on PVC` flow. Its not called for `OSD on Disks` flow in `OSD prepare pod`.  (To be more precise `initializeDevices` method does not call `updateLVMConfig` method).  Is that expected ?

Reply:
You're right, I believe `initializeDevices` should also call `updateLVMConfig` during OSD prepare. Good catch.